docs/index.md

```diff
@@ -760,27 +759,27 @@ Mocha supports the `err.expected` and `err.actual` properties of any thrown `Ass
     -r, --require <name>                    require the given module
     -s, --slow <ms>                         "slow" test threshold in milliseconds [75]
     -t, --timeout <ms>                      set test-case timeout in milliseconds [2000]
-    -u, --ui <name>                         specify user-interface (bdd|tdd|qunit|exports)
+    -u, --ui <name>                         specify user-interface (bdd|tdd|qunit|exports) (default: bdd)
     -w, --watch                             watch files for changes
     --check-leaks                           check for global variable leaks
     --full-trace                            display the full stack trace
-    --compilers <ext>:<module>,...          use the given module(s) to compile files
+    --compilers <ext>:<module>,...          use the given module(s) to compile files (default: )
```

Comment:
Don't understand why you merged this with the obvious fix not done.
Add `[]` back as the default value.

Reply:
Why did you leave comments here? 
This PR was to update doc because it is outdated. So, we found `[]` problem here. 
I will update them is #3529 .